src/core/database/entities/MosaicModel.ts

```diff
@@ -49,6 +50,7 @@ export class MosaicModel {
         this.transferable = mosaicInfo.isTransferable();
         this.supplyMutable = mosaicInfo.isSupplyMutable();
         this.restrictable = mosaicInfo.isRestrictable();
```

Comment:
To make it consistent, we should change it to `flags.transferable` instead of using the `isTransferable`?
what do you think about it?

Reply:
Yes. I changed it.